[FEAT]: Improved the profile card by adding incomplete issues and total number of PR's

Aashiq-Edavalapati opened the pull request:
Fixes #20 

This PR implements the following changes:
1. Updated API to fetch completed and incomplete issues, separately.
2. Updated profile card UI by showing incomplete issues and total PR's also

### Demo
Here's a preview after the changes:
![image](https://github.com/user-attachments/assets/bceba747-579c-4ed1-9856-816acff6da6d)

Here are the pictures of the tables in the sample database:
1. **BountyLog table:**
![image](https://github.com/user-attachments/assets/e6f85672-6f91-443e-9d07-88c20a252142)

2.**Issue table:**
![image](https://github.com/user-attachments/assets/b86441e1-8f89-49bd-8f1c-d2dbe731d63c)

3.**Participant table:**
![image](https://github.com/user-attachments/assets/a97bfa67-c4b1-449b-bcec-7f25c60e8136)

4.**Project table:**
![image](https://github.com/user-attachments/assets/457f3f93-ab04-4b5b-940a-8f6bbb97b0bb)

5.**Solution table:**
![image](https://github.com/user-attachments/assets/5fc71aae-e1c2-45bf-b5de-aa02f1f05dd7)



### Responsive layout after changes:
1. **1024px width:**
![image](https://github.com/user-attachments/assets/8b718507-9909-4fd7-a173-1f5fc738514b)

2. **768px:**
![image](https://github.com/user-attachments/assets/7f060c75-1239-4e25-8ab3-c810d06791d0)

3. **425px:**
![image](https://github.com/user-attachments/assets/40b76883-24c2-45c3-8db6-e75afe279827)

4. **375px:**
![image](https://github.com/user-attachments/assets/720f99e6-1d69-488f-ba8d-131102e72fe2)

5. **320px:**
![image](https://github.com/user-attachments/assets/4f835a17-9100-4578-b4e1-7f727b947ad9)

Aashiq-Edavalapati commented:
Btw, @IAmRiteshKoushik , @Ashrockzzz2003 , just to mention, I've seen the responsiveness of the webpage recently. I think there's something wrong with the layout in smaller devices. Leaderboard is getting displayed over the profile card.

IAmRiteshKoushik commented:
LGTM. How about you @Ashrockzzz2003 ?

Ashrockzzz2003 reviewed (commented):
Looks good.

@Aashiq-Edavalapati Does the UI problem still exist in mobile view. If yes, fix that too and we'll merge this one.

Aashiq-Edavalapati commented:
Yes, the problem still exists and I'll try to fix it

Ashrockzzz2003 commented:
@Aashiq-Edavalapati 

#34 seems relevant. Fix that too!

Ashrockzzz2003 commented:
Add screenshots after you're done!

Aashiq-Edavalapati commented:
@Ashrockzzz2003 I have made the changes. Could you review when you are free

Ashrockzzz2003 commented:
> @Ashrockzzz2003 I have made the changes. Could you review when you are free

yeah
but add screenshots.

Aashiq-Edavalapati commented:
I have edited the PR description and added images in it

Ashrockzzz2003 reviewed (commented):
What about the `320px` case? It looks off

Aashiq-Edavalapati commented:
> What about the `320px` case? It looks off

Ok I'll fix it

Aashiq-Edavalapati commented:
@Ashrockzzz2003 I have fixed it by dynamically calculating the margin based on height of the usercard. I have added more issues in the database to test whether the scaling is working properly or not. Here's a preview of it for 320px width:

![image](https://github.com/user-attachments/assets/b78be457-60c3-41bb-a4ce-493c7a30abda)

Ashrockzzz2003 reviewed (approved):
I enhanced the UI a bit more.
Thanks for the update @Aashiq-Edavalapati

Ashrockzzz2003 commented:
/bounty 75 @Aashiq-Edavalapati
